llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -411,6 +417,30 @@ std::string saveModuleIR(Module &M, int I, StringRef Suff) {
   return OutFilename;
 }
 
+bool isImportedFunction(const Function &F) {
+
+  if ( // Functions with definitions are not imported
```

Comment:
IMO these comments are not adding much, the code itself says the exact same thing the comments say, so we can remove the comments

Reply:
Comments removed.